src/components/SingleArticle.jsx

```diff
@@ -33,20 +36,23 @@ const SingleArticle = () => {
         <Container fluid className="single-article">
          <Row>
             <h2>{displayArticle.title}</h2>
-            <p>Author: {displayArticle.author} 
-            <br />Topic: {displayArticle.topic}</p>
+            <p><strong>Author:</strong> {displayArticle.author} 
+            <br />
+            <strong>Topic:</strong> {displayArticle.topic}</p>
         </Row>
         <Row>
-            <img className='article-img' src={displayArticle.article_img_url}/>
+            <img src={displayArticle.article_img_url}/>
         </Row>
         <Row>
             <p>{displayArticle.body}</p>
         </Row>
         <Row>
-            <Col><Button>👍{displayArticle.votes}</Button></Col>
-            <Col><Button>Show comments: {displayArticle.comment_count}</Button></Col>
+            <Col><button className='interact-buttons'>👍{displayArticle.votes}</button></Col>
+            <Col><button className='interact-buttons' onClick={showComments}>{isVisible ? 'Hide Comments' : 'Show Comments: '} {isVisible ? ' ' :`${displayArticle.comment_count}`}</button></Col>
         </Row>
         </Container>
+        {isVisible && <Comments article_id={article_id}/>}
+        
```

Comment:
So this still does the loading state when there are no comments, like on article 37. It's not a major issue, but you could account for it by conditionally rendering a little message to the user telling them there are no comments on this article. Of course that is up to you 🙂